src/java/org/igniterealtime/openfire/plugins/jmxhttpbridge/web/admin/ConfigServlet.java

```diff
@@ -130,8 +130,8 @@ protected void doPost(HttpServletRequest request, HttpServletResponse response)
     protected void setDefaultAttributes(HttpServletRequest request, HttpServletResponse response)
     {
         request.setAttribute("isJmxEnabled", JMXManager.isEnabled());
-        request.setAttribute("port", StatsWebServer.JMXHTTPBRIDGE_WEBSERVER_PORT.getValue());
-        request.setAttribute("secureport", StatsWebServer.JMXHTTPBRIDGE_WEBSERVER_PORT_SECURE.getValue());
+        request.setAttribute("port", StatsWebServer.JMXHTTPBRIDGE_WEBSERVER_PORT.getValue() == -1 ? null : StatsWebServer.JMXHTTPBRIDGE_WEBSERVER_PORT.getValue());
+        request.setAttribute("secureport", StatsWebServer.JMXHTTPBRIDGE_WEBSERVER_PORT_SECURE.getValue() == -1 ? null : StatsWebServer.JMXHTTPBRIDGE_WEBSERVER_PORT_SECURE.getValue());
```

Comment:
Should these be `< 0` tests to be consistent with the descriptions in the i18n / javadoc?